dms/views/dms_portal_templates.xml

```diff
@@ -91,18 +90,23 @@
                                     t-options="{'widget': 'date'}"
                                 />
                             </td>
-                            <td />
                         </tr>
                     </t>
                     <t t-if="dms_files">
                         <t t-foreach="dms_files" t-as="dms_file">
                             <tr class="tr_dms_file">
                                 <td>
-                                    <img
-                                        t-att-src="image_data_uri(dms_file.thumbnail)"
-                                        style="width:30px"
-                                    />
-                                    <span t-esc="dms_file.name" />
+                                    <a
+                                        t-attf-href="/my/dms/file/#{dms_file.id}/download?{{ keep_query() }}"
```

Comment:
Side topic (is this download tokenized and thus not able to be dowloaded by anyone?)

Reply:
Link keep parameters (`access_token` for example) and controller decide who can download it (https://github.com/OCA/dms/blob/12.0/dms/controllers/portal.py#L179-L203)

Reply:
Thanks for the info.